<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QWidget" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="L_PacketOut">
        <property name="text">
         <string>Packets Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="TB_PacketOut"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="L_PacketIn">
          <property name="text">
           <string>Packets In</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="TB_PacketIn"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="L_DebugLog">
        <property name="text">
         <string>Debug Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="TB_DebugLog"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
